src/modules/meals/countMeals.js

```diff
@@ -0,0 +1,7 @@
+const countMeals = () => {
+  const totalMeals = document.querySelector('.total-meals');
+  const tot = document.querySelector('#meals_list').childElementCount;
+  totalMeals.innerHTML = `(${tot}) meals`;
+};
+
```

Comment:
I think A counter function should cover all the edge cases you can think about. try to write counter function that can meals by itself rather than taking a calculated number from  the other function which is written by me.

Reply:
the counter function counts the number of children (meals) in the meals section. I think it depends on itself